src/core/CMakeLists.txt

```diff
@@ -911,3 +911,10 @@ else()
     endif()
 endif()
 #--------------------------------------------------------------------
+
+
+set(PCH_HEADERS ${HEADER_FILES})
+# Should not include the following file in PCH or we will get multiple definitions of NvOptimusEnablement and AmdPowerXpressRequestHighPerformance
```

Comment:
hmm those definitions should be in a cpp only the declarations should be in the header!?! @ResearchDaniel Why is it written like that? should be changed.

Reply:
No good reason :)